### ISSUE TITLE
Does this compile on FreeBSD

### ISSUE THREAD

**Issue by PhilZ-cwm6** [kevincox/rustymedia#22]
Hi,
Can you provide compile details ? Can the server compile and run on FreeBSD ?
Does it support subtitles when transcoding ? What formats are supported ?

Best regards

**Comment by kevincox** [comment]
> Can you provide compile details ?

https://github.com/kevincox/rustymedia#usage

> Can the server compile and run on FreeBSD ?

I have never tried. I am not aware of any reason why it wouldn't.

> Does it support subtitles when transcoding ?

IIRC extra streams like subtitles are passed through so it should work.

> What formats are supported ?

I think it shuold support just about anything that ffmpeg does. But IIRC there is a hardcoded (but easy to extend) extension list that it considers.

**Comment by PhilZ-cwm6** [comment]
Thank you a lot, I will test it and report back !
A last question: is there a config file in sources where we can modify ~~SMB~~ DLNA server settings: Name, icon, user/owner ?
If it works properly, I will try to add script to run it as a service under FreeBSD
Best regards

**Comment by kevincox** [comment]
No, it is really all hardcoded right now: https://github.com/kevincox/rustymedia/blob/master/src/dlna/root.xml

Also this does DLNA, not SMB.

**Comment by PhilZ-cwm6** [comment]
Sorry for the typo, yes, I meant DLNA
`friendlyName` and `iconList` xml tags can be used to customize icon and server name ? I know it is only cosmetics, but just asking before I have time to try it, probably in the next 2 weeks
Thank you again for the prompt support, as this looks interesting compared to heavy tools like Plex

**Comment by kevincox** [comment]
I really don't remember. I haven't used this in a long time.
